angrmanagement/ui/views/disassembly_view.py

```diff
@@ -104,6 +103,20 @@ def setFocus(self):
     # Properties
     #
 
+    @property
+    def caption(self):
+        return f'{self.base_caption}-{self.index}'
+
+    @caption.setter
+    def caption(self, cap):
```

Comment:
Let's get rid of `caption.setter`. `caption` should be a read-only property that is always constructed from `self.base_caption` and `self.index`. You'll want to add `self.base_caption = 'Disassembly` in `DisassemblyView.__init__` and `self.index = 1` in `BaseView.__init__`.

Also, Please move `caption` from `DisassemblyView` to `BaseView` so all views are treated equally.

Reply:
updated!